components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal.tsx

```diff
@@ -54,28 +54,25 @@ const modalMessages = defineMessages({
     },
 });
 
-const KeyboardShortcutsModal = (): JSX.Element => {
-    const [show, setShow] = useState<boolean>(false);
+interface Props {
+    onExited: () => void;
+}
 
-    useEffect(() => {
-        //toggles the state of shortcut dialog
-        const handleToggle = (): void => setShow(!show);
-        ModalStore.addModalListener(Constants.ActionTypes.TOGGLE_SHORTCUTS_MODAL, handleToggle);
-        return () => {
-            ModalStore.removeModalListener(Constants.ActionTypes.TOGGLE_SHORTCUTS_MODAL, handleToggle);
-        };
-    }, []);
+const KeyboardShortcutsModal: FC<Props> = (props: Props) => {
```

Comment:
We had some previous discussions around whether or not we should be using `FC` in cases like this since I think it does some weird things like always making the component support children even if they aren't actually used. I think we ended up deciding not to specify a type for functional components like this.

Reply:
ah will keep in mind, reverted